### PR TITLE
Update airlines.dat

### PR DESCRIPTION
**Pull request by diibify**
Complemented by AIRAC 2201/1

### CODE REVIEW

File: data/airlines.dat [hunk 1]
```diff
@@ -474,17 +483,19 @@ AXV	AVIAXESS	AXAVIA	FRANCE
...
+AYH	AVIOANDES S.A.	ANDES	EQUADOR
 AYL	ALPINE AIR PRIVATE LIMITED	ALPINE NEPAL	NEPAL
 AYM	AIRMAN, S.L.	AIRMAN	SPAIN
 AYN	ATLANTIC AIRLINES, S.A.	ATLANTIC NICARAGUA	NICARAGUA
 AYR	FLIGHT TRAINING EUROPE S.L.	CYGNET	SPAIN
 AYT	AYIT AVIATION & TOURISM	AYEET	ISRAEL
 AYV	AIRWAYS SERVICES & SUPPORT AS&S, C.A.	CARGWAY	VENEZUELA
 AYY	AIR ALLIANCE EXPRESS AG & CO. KG, BURBACH	LUPUS	GERMANY
-AZA	ALITALIA SOCIETÀ AEREA ITALIANA S.P.A.	ALITALIA	ITALY
+AZA	ALITALIA SOCIET├Ç AEREA ITALIANA S.P.A.	ALITALIA	ITALY
```
Comment (jonaseberle) [inline data/airlines.dat:498]: something's odd here with the encoding

File: data/airlines.dat [hunk 2]
```diff
@@ -5944,14 +6157,15 @@ VPB	VETERAN AIR	VETERAN	UKRAINE
...
+VPT	STREAM AIR	POTOK	RUSSIAN FEDERATION
 VPV	VIP-AVIA	VIP AVIA	GEORGIA
 VPX	AIRCRAFT VIP SERVICES	TOPLUX	LUXEMBOURG
 VQI	VILLA AIR PRIVATE LIMITED	VILLA AIR	MALDIVES
 VRA	VERITAIR LTD	VERITAIR	UNITED KINGDOM
 VRB	SILVERBACK CARGO FREIGHTERS	SILVERBACK	RWANDA
 VRC	TAXI AEREO DE VERACRUZ	VERACRUZ	MEXICO
 VRD	VIRGIN AMERICA, INC (BURLINGAME, CA)	REDWOOD	UNITED STATES
-VRE	AIR COTE D' IVOIRE	COTE D'IVOIRE	CÔTE D'IVOIRE
+VRE	AIR COTE D' IVOIRE	COTE D'IVOIRE	C├öTE D'IVOIRE
```
Comment (jonaseberle) [inline data/airlines.dat:6168]: here too

**Review by jonaseberle — CHANGES_REQUESTED** [review]

**Comment by jonaseberle** [comment]
Thank you! 
A lot of additions, that's great :) We just have to double check the special chars ô Â etc.

**Comment by diibify** [comment]
Damn Microsoft. See the next request - I hope all is fine for now.

**Comment by jonaseberle** [comment]
You can update pull requests as welll ;) If you want I can show you. Are you also in the Swift discord?

**Pull request closed without merging**